apollo-federation/tests/composition/supergraph_reversibility.rs

```diff
@@ -0,0 +1,202 @@
+use apollo_federation::Supergraph;
+use apollo_federation::subgraph::typestate::Subgraph;
+use apollo_federation::utils::normalize_schema::normalize_schema;
+
+use super::ServiceDefinition;
+use super::compose_as_fed2_subgraphs;
+
+fn compose_and_test_reversibility(subgraphs: &[ServiceDefinition<'_>]) {
+    let result = compose_as_fed2_subgraphs(subgraphs).unwrap();
+
+    let actual_subgraphs = Supergraph::new(&result.schema().schema().to_string())
+        .unwrap()
```

Comment:
Global comment: Use expect() instead of unwrap()?

Reply:
Makes sense, switched the `.unwrap()`s to `.expect()`s.